app/src/main/java/com/babylon/wallet/android/domain/RadixWalletException.kt

```diff
@@ -471,7 +471,7 @@ fun Throwable.getDappMessage(): String? {
             "Wallet is using network ID: $currentNetworkId, request sent specified network ID: $requestNetworkId"
         }
 
-        else -> message
+        else -> message ?: cause?.message
```

Comment:
Some errors carry a cause from the `CommonException` coming from sargon. This message can be passed back to the CE logs.